Warning Points

DubStepMad opened the pull request:
Allows warning points to be set in the players table with the same colum (warning) used in 'notes'. This does not create the colum in the 'players' table.

Other edit allows the warning points to be displayed on the users account information.

DubStepMad committed:
Update notes.php

DubStepMad committed:
Update editPlayer.php

DubStepMad committed:
Add files via upload

Adds a case logging & log page, Uses a modified reimbursement_log table which uses the statement found here: https://pastebin.com/vBbV11qD

DubStepMad committed:
Update notes.php

Fix for warning pointsnot being added and error 'column can not be null' has been fixed.

Jason2605 commented on notes.php:
```diff
@@ -81,16 +81,16 @@
     echo '<td>'."<input class='form-control' type=text name=note value=''> </td>";
     echo '<td>'."<input class='btn btn-primary btn-outline' type=submit name=update value=Update".'> </td>';
-    echo "<td style='display:none;'>".'<input type=hidden name=hidden value='.$row['uid'].'> </td>';
+    echo "<td style='display:none;'>".'<input type=hidden name=hidden value='.$row['playerid'].'> </td>';
```
I prefer the use of UID here over playerid due to 5.0 changing the table to be pid.

DubStepMad commented:
I will try later when I get home to update it with the UID.

On Wed, May 3, 2017 at 10:29 PM, Jason_000 <notifications@github.com> wrote:

> *@Jason2605* commented on this pull request.
> ------------------------------
>
> In notes.php
> <https://github.com/Jason2605/AdminPanel/pull/28#discussion_r114658339>:
>
> >      echo '<td>'."<input class='form-control' type=text name=note value=''> </td>";
>      echo '<td>'."<input class='btn btn-primary btn-outline' type=submit name=update value=Update".'> </td>';
> -    echo "<td style='display:none;'>".'<input type=hidden name=hidden value='.$row['uid'].'> </td>';
> +    echo "<td style='display:none;'>".'<input type=hidden name=hidden value='.$row['playerid'].'> </td>';
>
> I prefer the use of UID here over playerid due to 5.0 changing the table
> to be pid.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jason2605/AdminPanel/pull/28#pullrequestreview-36143738>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANKRsLEV3hbpULLxhvRNdgYfXpiiiEQ6ks5r2PGngaJpZM4NFafn>
> .
>

DubStepMad committed:
Update notes.php

Allows the use of uid instead of playerid. Since all dbs use uid to link a player, there is no need to use playerid unless a new framework uses it.

Jason2605 commented on notes.php:
```diff
@@ -80,8 +80,8 @@
     echo '<td>'.$row['name'].' </td>';
-    echo '<td>'.$row['aliases'].' </td>';
-    echo '<td>'."<select class='form-control' name='warn'><option value='4'>Commendation</option><option value='1' selected='selected'>Warning</option><option value='2'>Caution</option><option value='3'>Big Caution</option></select> </td>";
+    echo '<td>'.$row['playerid'].' </td>';
```
Same here

DubStepMad commented:
I added row 83 to it for my own use, can be removed since it is not really
needed.

Connor

On 10 May 2017 18:39, "Jason_000" <notifications@github.com> wrote:

> *@Jason2605* commented on this pull request.
> ------------------------------
>
> In notes.php
> <https://github.com/Jason2605/AdminPanel/pull/28#discussion_r115804879>:
>
> > @@ -80,8 +80,8 @@
>      echo '<tr>';
>      echo '<td>'.$row['uid'].' </td>';
>      echo '<td>'.$row['name'].' </td>';
> -    echo '<td>'.$row['aliases'].' </td>';
> -    echo '<td>'."<select class='form-control' name='warn'><option value='4'>Commendation</option><option value='1' selected='selected'>Warning</option><option value='2'>Caution</option><option value='3'>Big Caution</option></select> </td>";
> +    echo '<td>'.$row['playerid'].' </td>';
>
> Same here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jason2605/AdminPanel/pull/28#pullrequestreview-37376388>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANKRsL6IMGBafIPPXBfrBPhvR5RGKQQTks5r4fY6gaJpZM4NFafn>
> .
>

DubStepMad committed:
Update notes.php

Removal of playerid  being used (Still there on the concat search) and added so that the current warning points can be viewed.

DubStepMad committed:
Update notes.php

DubStepMad committed:
Warning system & Staff Notes

Creates the warning column in the players table and adds a new table to use the for the staff notes. It uses the notes table already for the warning .

itsJarrett commented:
Very interesting concept.

itsJarrett commented:
@Jason2605 might want to make your decisions whether or not to include this. Other contributions to the repo are creating conflicts. 🕐
